cloudsql-postgresql-plugin/src/e2e-test/java/io/cdap/plugin/cloudsqlpostgresql/stepsdesign/CloudSqlPostgreSql.java

```diff
@@ -0,0 +1,73 @@
+package io.cdap.plugin.cloudsqlpostgresql.stepsdesign;
+
+import io.cdap.e2e.pages.actions.CdfPipelineRunAction;
+import io.cdap.e2e.utils.BigQueryClient;
+import io.cdap.e2e.utils.CdfHelper;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import io.cdap.plugin.cloudsqlpostgresql.BQValidation;
+import io.cdap.plugin.cloudsqlpostgresql.CloudSqlPostgreSqlClient;
+import io.cucumber.java.en.Then;
+import org.junit.Assert;
+import stepsdesign.BeforeActions;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.text.ParseException;
+
+/**
+ *  CLOUDSQLPOSTGRESQL Plugin related step design.
+ */
+public class CloudSqlPostgreSql implements CdfHelper {
+
+  @Then("Click on preview data for CloudSQLPostgreSQL sink")
```

Comment:
Why do we need to write a step here, we can use it from framework.

Reply:
Done.